discovery-provider/src/queries/query_helpers.py

```diff
@@ -1178,7 +1178,7 @@ def seconds_ago(timestamp):
     return func.extract("epoch", (func.now() - timestamp))
 
 
-def decayed_score(score, created_at, peak=5, nominal_timestamp=14 * 24 * 60 * 60):
+def decayed_score(score, created_at, peak=100, nominal_timestamp=60 * 24 * 60 * 60):
```

Comment:
peak of 1000 looks better for both tracks and playlists after running a couple tests IMO:

![Screen Shot 2021-07-14 at 5 37 16 PM](https://user-images.githubusercontent.com/2731362/125710036-a80d47f3-8fb9-4af5-a07a-da17f3261b70.png)
![Screen Shot 2021-07-14 at 5 37 27 PM](https://user-images.githubusercontent.com/2731362/125710045-33411cbf-fac7-40b3-b9dc-045285d6ddca.png)

Reply:
nice 👍